### PR TITLE
add logic to manage registers in num operation

### PR DESCRIPTION
**Pull request by skuater**
With this fix u can work with registers not flaged and alias register in aritmetic operations:
  aer PC=PC+1
  aer eip=PC+1
Before this fix u can work with this stuff if registers are flagged, but dont support register aliases.

### CODE REVIEW

**Comment by radare** [comment]
One broken test

**Comment by radare** [comment]
Ping

**Comment by radare** [comment]
Please fix this for the release

**Comment by radare** [comment]
REPING

File: libr/core/core.c [hunk 1]
```diff
@@ -572,6 +572,22 @@ static ut64 num_callback(RNum *userptr, const char *str, int *ok) {
 				ret = flag->offset;
 				if (ok) *ok = true;
 			}
+			// check for reg alias
+			/*struct r_reg_item_t *r = r_reg_get (core->dbg->reg, str, -1);
```
Comment (radare) [inline libr/core/core.c]: lol

**Comment by radare** [comment]
Ping

**Comment by radare** [comment]
pls update/fix

**Comment by skuater** [comment]
This was pulled few days ago with the aea fix

**Pull request closed without merging**